### DISCUSSION TITLE
[LocalSpan] Why doesn't CorrelationContext perform write-back? or how do it

### DISCUSSION THREAD

**Discussion by darknesstm** [apache/skywalking#10719] [General]
now CorrelationContext is copied in the current cross-thread design, changes made to it in another thread will not be reflected back to the originating thread. 

Is there any way to perform write-back currently?

**Comment by wu-sheng** [comment]
CorrelationContext is a one-way pushing-forward context, rather than a memory-sharing mechanism.
